Documentation/ceph-disaster-recovery.md

```diff
@@ -94,9 +94,10 @@ ceph-mon \
 
 (be sure to remove the single quotes around the `--log-stderr-prefix` flag)
 
-Patch the `rook-ceph-mon-b` Deployment to run a sleep instead of the `ceph mon` command:
+Patch the `rook-ceph-mon-b` Deployment to stop livenessProbe and run a sleep instead of the `ceph mon` command:
```

Comment:
> Patch the `rook-ceph-mon-b` Deployment to stop livenessProbe and run a sleep instead of the `ceph mon` command:

It says what two commands do as it. How about the following sentence to explain why we should run that commands?

```
Patch the `rook-ceph-mon-b` Deployment to stop this mon working without deleting the mon pod.
```

Reply:
Thanks for your suggestion. I will fix it.